### PR TITLE
tweak reply_to logic

### PR DESCRIPTION
**Pull request by erikolsson**
i think we should do something like this to prevent notifications from always displaying "replied to you" inside space channels

### CODE REVIEW

**Review by andreyz — APPROVED** [review]
Nice!

**Review by bas-vk — APPROVED** [review]

**Review by bas-vk — APPROVED** [review]

**Review by texuf — COMMENTED** [review]
seems a little weird?

**Comment by texuf** [comment]
> seems a little weird? 

Sorry I didn't submit my comments

File: core/node/notifications/event_processor.go [hunk 1]
```diff
@@ -164,9 +164,10 @@ func (p *MessageToNotificationsProcessor) OnMessageEvent(
 			recipients.Add(participant)
 		case *StreamEvent_ChannelPayload:
 			if spaceID != nil {
-				if p.onSpaceChannelPayload(*spaceID, channelID, participant, pref, event) {
+				sendNotification, notificationKind := p.onSpaceChannelPayload(*spaceID, channelID, participant, pref, event)
+				if sendNotification {
 					usersToNotify[participant] = pref
-					kind = "reply_to"
+					kind = notificationKind
```
[the remark this answers is not in the document]
Comment (texuf) [reply core/node/notifications/event_processor.go:170]: oh, i see - the participating logic should go into WantNotificationForSpaceChannelMessage, the conversion from type to kind is 1 to 1, should go in its own function

**Review by texuf — APPROVED** [review]
💯

**Pull request merged**